src/components/Game.tsx

```diff
@@ -0,0 +1,65 @@
+import { type } from "@testing-library/user-event/dist/type";
+import { useMemo, useState } from "react";
+import Board from "./Board";
+
+function Game() {
+  const resetbox: Array<string> = Array(9).fill("");
+  const [xIsNext, setxIsNext] = useState<boolean>(true);
+  const [box, setBox] = useState<Array<string>>(resetbox);
+
+  const handleClick = (index: number) => {
+    if (calculateWinner(box) || box[index]) {
+      return;
+    }
+    box[index] = xIsNext ? "X" : "O";
+    setBox(box);
+    setxIsNext(!xIsNext);
+  };
+
+  const winner = useMemo(() => {
+    const turn = calculateWinner(box);
+    if (turn) {
+      return turn + " is Winner!";
+    } else if (!turn && !box.includes("")) {
```

Comment:
I think we can omit this since `turn` can not be true at this point.
```suggestion
    } else if (!box.includes("")) {
```